dev/workshop-syllabus.md

```diff
@@ -42,25 +42,22 @@ Sudents will be expected to participate in the workshop in a hands-on way, follo
 |----------------------------------------|------|
 | Data import and quality control        | 30m  |
 | Data preprocessing and visualisation   | 30m  |
-| Differential transcript abundance      | 30m  |
+| Differential gene abundance            | 30m  |
 | Data visualisation                     | 30m  |
 
 # Workshop goals and objectives
 
-In exploring and analysing RNA sequencing data, there are recurrent concept such as filtering, scaling, dimensionality reduction, hypothesis test, clustering and visualisation. These concepts can be intuitively explained to new users, however (i) the heterogeneity of the vocabulary across methodologies/algorithms/packages, (ii) the complexity of data wrangling, and (iii) the coding burden play against an effective learning experience of the statistics and biology underlying an informed data analysis. 
+In exploring and analysing RNA sequencing data, there are a number of key concepts, such as filtering, scaling, dimensionality reduction, hypothesis testing, clustering and visualisation, that need to be understood. These concepts can be intuitively explained to new users, however, (i) the use of a heterogeneous vocabulary and jargon by methodologies/algorithms/packages, (ii) the complexity of data wrangling, and (iii) the coding burden, impede effective learning of the statistics and biology underlying an informed RNA sequencing analysis. 
 
-The tidyverse (tidytranscriptomics) approach to (RNA sequencing) data analysis abstracts out the coding related complexity and presents tools that use an intuitive and jargon-free vocabulary, so allowing to focus to the biological and the statistical challenges. Furthermore, it allows the compilation of modular and pipe oriented workflows that allow new user to compare alternative analysis algorithms and workflows.
+The tidytranscriptomics approach to RNA sequencing data analysis abstracts out the coding-related complexity and provides tools that use an intuitive and jargon-free vocabulary, enabling focus on the statistical and biological challenges.
 
 ## Learning goals
 
-* To understand of the recurrent concepts and steps of bulk RNA sequencing data analysis
-* To approach data representation and analysis though a tidy paradigm, integrating tidyverse with tidybulk.
+* To understand the key concepts and steps of bulk RNA sequencing data analysis
+* To approach data representation and analysis though a tidy data paradigm, integrating tidyverse with tidybulk.
 
 ## Learning objectives
 
-* To remember the recurrent concept and vocabulary of RNA sequencing data analysis
```

Comment:
With this section they explicitly asked to follow their system , remember, understand, apply.. all points with key words. It was specified in the original document.

Reply:
As far as I understand it those are just examples they have provided, they just want us to include some e.g. they say "Some examples:" in the original template. Here's an example of the plyranges workshop syllabus from last year: https://github.com/sa-lee/plyrangesWorkshops